github/github.go

```diff
@@ -154,32 +155,32 @@ type Client struct {
 	common service // Reuse a single struct instead of allocating one for each service on the heap.
 
 	// Services used for talking to different parts of the GitHub API.
-	Actions        *ActionsService
-	Activity       *ActivityService
-	Admin          *AdminService
-	Apps           *AppsService
-	Authorizations *AuthorizationsService
-	Billing        *BillingService
-	Checks         *ChecksService
-	CodeScanning   *CodeScanningService
-	Enterprise     *EnterpriseService
-	Gists          *GistsService
-	Git            *GitService
-	Gitignores     *GitignoresService
-	Interactions   *InteractionsService
-	IssueImport    *IssueImportService
-	Issues         *IssuesService
-	Licenses       *LicensesService
+	Actions        ActionsServiceInterface
+	Activity       ActivityServiceInterface
+	Admin          AdminServiceInterface
+	Apps           AppsServiceInterface
+	Authorizations AuthorizationsServiceInterface
+	Billing        BillingServiceInterface
+	Checks         ChecksServiceInterface
+	CodeScanning   CodeScanningServiceInterface
+	Enterprise     EnterpriseServiceInterface
+	Gists          GistsServiceInterface
+	Git            GitServiceInterface
+	Gitignores     GitignoresServiceInterface
+	Interactions   InteractionsServiceInterface
+	IssueImport    IssueImportServiceInterface
+	Issues         IssuesServiceInterface
+	Licenses       LicensesServiceInterface
 	Marketplace    *MarketplaceService
```

Comment:
Just a reminder: There's a bit of work needed on https://github.com/gmlewis/go-github/blob/master/github/apps_marketplace_test.go#L62 before changing this to also use the interface. I tried to understand why that property is needed but I guess I don't have enough context on why.

Reply:
> I tried to understand why that property is needed but I guess I don't have enough context on why.

Here is the context: https://github.com/google/go-github/pull/732#issuecomment-333726743

Reply:
I will have a read, thanks!

Reply:
After reading on that issue and also having a quick read on GitHub's Api, would there be any objections on adding `get ` and `set` methods for `s.Stubbed`? This would be a small change and we would still be able to use the interfaces as long as we regenerate `github-interfaces.go`. I guess the other option would be a switch type inside some of the methods from `MarketplaceService` to distinguish between the real implementation and the fake ones, but I'm not sure if that's preferable over the simple `get` and `set`.

Reply:
This is PR with the getter and setter implementation: https://github.com/gmlewis/go-github/pull/2. Let me know your thoughts. :+1:

Reply:
> This is PR with the getter and setter implementation: [gmlewis#2](https://github.com/gmlewis/go-github/pull/2). Let me know your thoughts.

That's probably the best way to proceed.

Reply:
Stubbed methods incorporated.  Resolving conversation.